source/connectors/kafka-sink-properties.txt

```diff
@@ -0,0 +1,249 @@
+.. _kafka-sink-properties:
+
+=============================================
+Kafka Sink Connector Configuration Properties
+=============================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Sink Connector Configuration Properties
+---------------------------------------
+
+Refer to the table below to create or migrate a properties file for your
+MongoDB Kafka sink connector. See `MongoSinkConnector.properties
+<https://github.com/mongodb/mongo-kafka/blob/master/config/MongoSinkConnector.properties>`_
+for an example configuration file.
```

Comment:
1) Should I leave "migrate" out? The migration instructions were in the prior section. I thought this section could enforce the structure in case there were any other changes that were undocumented.

2) Thanks, will add.

3) Sure, will change. I got mixed up by some other example I looked at, so will make sure to update other instances as I encounter.